### DISCUSSION TITLE
How to view point spectra on simultaneously obtained topograph

### DISCUSSION THREAD

**Discussion by YuF12345** [yuksk/SIDAM#45] [General]
As is seen in Gif movies, spectrum viewer, it seems possible to view each point spectrum on simultaneously obtained topograph.
Could you tell me how make it possible?

**Comment by yuksk** [comment]
Select "Target window" from the menu (≡) of the window where a spectrum is displayed, and you will find a list of windows. Select a window from the list and move the mouse over the window to view the spectrum at the mouse position.

**Reply by YuF12345** [reply]
I could do it.
What a fantastic macro! Thank you so much.
